### DISCUSSION TITLE
Volume controls

### DISCUSSION THREAD

**Discussion by kj20029** [OnionUI/Onion#905] [Ideas]
Is it possible to add volume controls in the setting to increase and decrease the volume, I have a Miyoo mini+ and sometimes I get issue that I can’t control the volume so it would help to see if is a software issue or hardware, thanks for the OS it’s just great on Miyoo mini+

**Comment by Aemiii91** [comment]
You can edit the volume value in `/appconfigs/system.json` using the File Explorer app.
